src/index.html

```diff
@@ -6,27 +6,27 @@
     <title>Whats Cookin</title>
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,800&display=swap' rel='stylesheet'>
     <link rel="page" href="./pantry.html">
-    <link rel="stylesheet" href="/styles.css">
+    <link rel="stylesheet" href="./css/styles.css">
   </head>
   <body>
     <header class='main-header'>
       <div class="wrapper">
         <img class="hamburger-menu" src="/hamburger-icon.png" alt="hamburger icon serving as menu button">
-        <img class="x-menu hidden" src="/x-icon.png" alt="x icon to be clicked when you are done viewing the menu options">
+        <img class="x-menu hidden" src="/images/x-icon.png" alt="x icon to be clicked when you are done viewing the menu options">
       </div>
       <h1 class="header-title">What's Cookin'?</h1>
       <section class="user-info">
        <p class='user-name'> Welcome</p>
       </section>
     </header>
-    <section class="open-menu-selections x">
-      <article class="menu-my-pantry menu-selection x">
+    <section class="open-menu-selections hidden">
+      <article class="menu-my-pantry menu-selection hidden">
         <h2 class="menu-my-pantry-title"><a href="./pantry.html">My Pantry</a></h2>
       </article>
-      <article class="menu-my-upcoming-recipes menu-selection">
+      <article class="menu-my-upcoming-recipes menu-selection hidden">
         <h3 class="menu-my-upcoming-recipes-title">My Upcoming Recipes</h2>
       </article>
-      <article class="menu-favorites menu-selection ">
+      <article class="menu-favorites menu-selection hidden">
```

Comment:
Thanks for adding those `.hidden` classes to get our initial display up with a closed menu!